src/sentry/static/sentry/app/components/acl/featureDisabled.jsx

```diff
@@ -31,12 +31,13 @@ class FeatureDisabled extends React.Component {
      */
     featureName: PropTypes.string,
     /**
-     * Render the disabled message within a warning Alert.
+     * Render the disabled message within a warning Alert. A custom Alert
+     * component may be provided.
      *
-     * Attaches additional styles to the FeatureDisabeld component to make it
-     * look nice within the Alert.
+     * Attaches additional styles to the FeatureDisabled component to make it
+     * look consistent within the Alert.
      */
-    alert: PropTypes.bool,
+    alert: PropTypes.oneOfType([PropTypes.bool, PropTypes.func]),
```

Comment:
this is kind of a confusing combination of acceptable values. Maybe instead something like:

```js
static propTypes = {
  //other stuff
  container: PropTypes.element
  alert: PropTypes.bool
}
```

would work?

Reply:
I'm okay with this I think